bin/utils/node-utils.sh

```diff
@@ -149,10 +149,10 @@ check_node_version() {
   
   too_low=""
   too_low_support=""
-  if [[ ${node_major_version} -lt 8 ]]
+  if [[ ${node_major_version} -lt 8 ]] || [[ ${node_major_version} -lt 12 ]]
```

Comment:
This is wrong because,
1) 8 is also -lt 12 :)
2) min version is 14, not 12.

why not just set this as a number at the top of the file, and reference it.
minVersion=14

if [[ ${node_major_version} -lt ${minVersion} ]]